src/main/java/com/zendesk/maxwell/schema/Table.java

```diff
@@ -142,7 +142,7 @@ private void diffColumnList(List<String> diffs, Table a, Table b, String nameA,
 					EnumeratedColumnDef enumA, enumB;
 					enumA = (EnumeratedColumnDef) column;
 					enumB = (EnumeratedColumnDef) other;
-					if ( !Arrays.deepEquals(enumA.getEnumValues(), enumB.getEnumValues()) ) {
+					if ( !enumA.getEnumValues().equals(enumB.getEnumValues()) ) {
```

Comment:
I'm not sure I remember or understand why `deepEquals` was used here, or why the change...

Reply:
`Arrays.deepEquals` is supposed to be used for nested arrays, which this isn't. It probably should've been `Arrays.equals` from the beginning. You can't check `(String[]a).equals((String[]b)` or use `==` so you need one of the Array equals helper methods. With this changed to a (immutable) List to avoid allowing the array to be mutated, we need to use the normal equals method again.